service/src/main/java/com/ge/predix/acs/audit/PredixEventMapper.java

```diff
@@ -22,9 +23,11 @@
 @Profile("predixAudit")
 public class PredixEventMapper {
 
-    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(PredixEventProcessor.class);
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(PredixEventMapper.class);
 
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final Pattern REGEX = Pattern
+            .compile("([0-9a-fA-F]{8})([0-9a-fA-F]{4})([0-9a-fA-F]{4})" + "([0-9a-fA-F]{4})([0-9a-fA-F]+)");
```

Comment:
since [0-9a-fA-F] is repeated at many places , i would create a separate constant for it and use it, rather than repeating it as well as it will regex more readable imo